### PR TITLE
[Node SDK] 'ru' locale for NodeJs BotBuilder

### PR DESCRIPTION
**Pull request by hardsky**
Hi!
For my project I added 'ru' localization for bot.
I want to share it with community, I hope it will be helpful for someone else too.

issue in tracker #2084

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@hardsky__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by hardsky** [comment]
Ok, signed.

**Comment by alexsorokoletov** [comment]
FYI, for Russian localization we are also overriding `EntityRecognizer.yesExp`/`EntityRecognizer.noExp` properties to make sure SDK recognizes different yes/no alternatives in Russian language.

```
builder.EntityRecognizer.yesExp = /^(1|y|yes|yep|sure|ok|true|да|ды|так|конечно|подтверждаю|согласен|ага|верно)/i
builder.EntityRecognizer.noExp = /^(2|n|no|nope|not|false|нет|неа|не)/i
```
Would be great to move these values to the localization JSON files and have everything in one place. But that's an SDK task.

**Comment by hardsky** [comment]
It's good point about EntityRecognizer!

**Comment by hardsky** [comment]
Actually, there is a place where used localized answers.
Look at `confirm` sources
```
        args.enumValues = [
            session.localizer.gettext(locale, 'confirm_yes', consts.Library.system),
            session.localizer.gettext(locale, 'confirm_no', consts.Library.system)
        ];
```
`var best = findChoice(args, text);`
at SimplePromptRecognizer.recognize

I tested with ru locale and it recognized answer 'да' as true.

**Comment by alexsorokoletov** [comment]
But it will not recognize ага as positive until you change yesExp

Alex

File: Node/core/lib/locale/ru/BotBuilder.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "default_number": "Не похоже на число. Пожалуйста, введите число.",
+    "default_confirm": "Не понимаю. Пожалуйста, ответьте 'да' или 'нет'.",
+    "list_or": " или ",
+    "list_or_more": ", или ",
+    "confirm_yes": "да",
+    "confirm_no": "нет",
+    "default_choice": "Не понимаю. Пожалуйста, выберите ответ из списка.",
+    "default_time": "Не похоже на время. Пожалуйста, повторите в формате (MM/ДД/ГГГГ ЧЧ:MM:СС).",
+    "default_file": "Файл не получен. Пожалуйста, повторите.",
+    "default_error": "Не предвиденная ошибка. Мы вынуждены начать сначала."
```
Comment (enby) [inline Node/core/lib/locale/ru/BotBuilder.json]: непредвиденная

Comment (hardsky) [inline Node/core/lib/locale/ru/BotBuilder.json]: Thanks! Added fix.

**Comment by hardsky** [comment]
fixed

**Comment by Stevenic** [comment]
Thanks!

**Pull request merged**